### ISSUE TITLE
Add proposition metrics

### ISSUE THREAD

**Issue by salilkanetkar** [adobe/xdm#233]
## What are the schemas that are affected by the issue
[Metrics](https://github.com/adobe/xdm/blob/master/schemas/data/metrics.schema.json)

## What are examples of products that are impacted by the issue
As such, none of the existing products will be affected. This issue will help meet the needs of offer management use-cases, where an application `proposes` an offer to another solution.

**Comment by trieloff** [comment]
Makes sense, but will need more fleshing out.

**Comment by cdegroot-adobe** [comment]
This does not make sense to me in ExperienceEvent. ExperienceEvent is centric about the action a user does and the state of the experience at that moment. So we may well have metrics like 'product-views' that were impacted by a 'proposition' but the user did not do a 'proposition' a system did it that the user has no direct control over. 

There needs to be a new schema created that represents the decision around a proposition.

**Comment by trieloff** [comment]
Agree, this should not be an ExperienceEvent, but modeled through EventEnvelope, so that it can still be observed.
